test/float8/test_base.py

```diff
@@ -8,9 +8,9 @@
 import itertools
 import random
 import re
-from typing import List, Tuple
 import unittest
 import warnings
+from typing import List, Tuple
```

Comment:
nit: it's ok for this PR, but in general it's good to do formatting changes in separate PRs, since this is not related to the change you're trying to accomplish.  If you do end up doing this, good practice to comment and just say something like "this is unrelated, just linting, etc".